proposals/simd/BinarySIMD.md

```diff
@@ -27,143 +27,170 @@ The `v8x16.shuffle` instruction has 16 bytes after `simdop`.
+| -                         |    `0x43`| -                  |
+| -                         |    `0x44`| -                  |
+| `f32x4.eq`                |    `0x45`| -                  |
+| `f32x4.ne`                |    `0x46`| -                  |
+| `f32x4.lt`                |    `0x47`| -                  |
+| `f32x4.gt`                |    `0x48`| -                  |
+| `f32x4.le`                |    `0x49`| -                  |
+| `f32x4.ge`                |    `0x4a`| -                  |
+| `f64x2.eq`                |    `0x4b`| -                  |
+| `f64x2.ne`                |    `0x4c`| -                  |
+| `f64x2.lt`                |    `0x4d`| -                  |
+| `f64x2.gt`                |    `0x4e`| -                  |
+| `f64x2.le`                |    `0x4f`| -                  |
+| `f64x2.ge`                |    `0x50`| -                  |
+| `i8x16.add`               |    `0x51`| -                  |
+| `i8x16.add_saturate_s`    |    `0x52`| -                  |
+| `i8x16.add_saturate_u`    |    `0x53`| -                  |
+| `i8x16.sub`               |    `0x54`| -                  |
+| `i8x16.sub_saturate_s`    |    `0x55`| -                  |
+| `i8x16.sub_saturate_u`    |    `0x56`| -                  |
+| `i8x16.mul`               |    `0x57`| -                  |
+| -                         |    `0x58`| -                  |
+| -                         |    `0x59`| -                  |
+| `i8x16.shl`               |    `0x5a`| -                  |
+| `i8x16.shr_s`             |    `0x5b`| -                  |
+| `i8x16.shr_u`             |    `0x5c`| -                  |
+| `i8x16.neg`               |    `0x5d`| -                  |
+| `i8x16.any_true`          |    `0x5e`| -                  |
+| `i8x16.all_true`          |    `0x5f`| -                  |
+| `i16x8.add`               |    `0x60`| -                  |
+| `i16x8.add_saturate_s`    |    `0x61`| -                  |
+| `i16x8.add_saturate_u`    |    `0x62`| -                  |
+| `i16x8.sub`               |    `0x63`| -                  |
+| `i16x8.sub_saturate_s`    |    `0x64`| -                  |
+| `i16x8.sub_saturate_u`    |    `0x65`| -                  |
+| `i16x8.mul`               |    `0x66`| -                  |
+| -                         |    `0x67`| -                  |
+| -                         |    `0x68`| -                  |
+| `i16x8.shl`               |    `0x69`| -                  |
+| `i16x8.shr_s`             |    `0x6a`| -                  |
+| `i16x8.shr_u`             |    `0x6b`| -                  |
+| `i16x8.neg`               |    `0x6c`| -                  |
+| `i16x8.any_true`          |    `0x6d`| -                  |
+| `i16x8.all_true`          |    `0x6e`| -                  |
+| `i32x4.add`               |    `0x6f`| -                  |
+| -                         |    `0x70`| -                  |
+| -                         |    `0x71`| -                  |
```

Comment:
Reserved for saturating adds.

Reply:
There are no hardware instructions that map to saturating arithmetic for i32x4, and i64x2 types that I know of, so I'm unsure if this will ever be useful - but the opcode space is large enough that this looks ok to me. Same for the other saturating operations below.